sources/czid-schema.graphql

```diff
@@ -208,7 +208,7 @@ type Query {
   pathogenList(version: String): PathogenList!
   project(id: Int!): Project!
   sample(sampleId: Int!): Sample!
-  sampleReadsStats(sampleIds: [Int!]!): SampleReadsStatsList!
+  sampleReadsStats(sampleIds: [String!]!): SampleReadsStatsList!
```

Comment:
is it valid to make this [ID!]?  or do we have to do this since it's being returned upstream as an array of strings?